docs/dev/git-cheat.md

```diff
@@ -0,0 +1,126 @@
+# Git cheat sheet for (busy) contributors
+
+## Requirement: DCO and signing commits
+
+Always DCO and sign your commits:
+
+```bash
+git -S -s -m "My commit"
+```
+
+Note: you do NOT need a gpg key to sign commits. You can just use your ssh key.
+
+Edit your `~/.gitconfig`:
+
+```
+[user]
+	name = me
+	email = myemail@example.com
+	signingkey = /myhome/.ssh/id_ed25519_sk.pub
```

Comment:
"_sk" seems unusual

Reply:
Right.
Using hardware securities key...
Probably same thing apply to edX.
Will make it look more familiar.